python_examples/rebound.py

```diff
@@ -1,4 +1,9 @@
 from ctypes import *
+import numpy
+import sys
```

Comment:
Similarly, I think we can do this without the package. How about

    try:
        xrange:
    except NameError:
        xrange = range